source/fundamentals/connection/connect.txt

```diff
@@ -61,146 +76,140 @@ a different dbpath, but all using the same replica set name "foo".
 
   $ mongod --port 27019 --dbpath /data/db2 --replSet foo
 
-Initializing the Set
-~~~~~~~~~~~~~~~~~~~~
+Your nodes are now running.
+
+Initializing and Connecting to the Replica Set
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-At this point all of our nodes are up and running, but the set has yet
-to be initialized. Until the set is initialized no node will become
-the primary, and things are essentially "offline".
+Before you can use your nodes, you must initialize your replica set and set one
+node as the primary node. Note that the following process is specific to uninitialized
+replica sets.
 
-To initialize the set we need to connect directly to a single node and run the
+To initialize the set, connect directly to a single node and run the
 initiate command using the ``directConnection`` option:
 
 .. code-block:: python
 
-  >>> from pymongo import MongoClient
-  >>> c = MongoClient('localhost', 27017, directConnection=True)
+   >>> from pymongo import MongoClient
+   >>> c = MongoClient('localhost', 27017, directConnection=True)
 
-.. note:: We could have connected to any of the other nodes instead,
-   but only the node we initiate from is allowed to contain any
+.. note:: You can connect to any of the node you created,
+   but only the node you initiate (in this case, the node running on port 27017) from is allowed to contain
    initial data.
 
-After connecting, we run the initiate command to get things started:
+After connecting to a single node, run the initiate command on all nodes:
 
 .. code-block:: python
 
-  >>> config = {'_id': 'foo', 'members': [
-  ...     {'_id': 0, 'host': 'localhost:27017'},
-  ...     {'_id': 1, 'host': 'localhost:27018'},
-  ...     {'_id': 2, 'host': 'localhost:27019'}]}
-  >>> c.admin.command("replSetInitiate", config)
-  {'ok': 1.0, ...}
+   >>> config = {'_id': 'foo', 'members': [
+   ...     {'_id': 0, 'host': 'localhost:27017'},
+   ...     {'_id': 1, 'host': 'localhost:27018'},
+   ...     {'_id': 2, 'host': 'localhost:27019'}]}
+   >>> c.admin.command("replSetInitiate", config)
+   {'ok': 1.0, ...}
 
-The three ``mongod`` servers we started earlier will now coordinate
-and come online as a replica set.
+The three ``mongod`` servers are now online as a replica set.
 
-Connecting to a Replica Set
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Connecting to an Existing Replica Set
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The initial connection as made above is a special case for an
-uninitialized replica set. Normally we'll want to connect
-differently. A connection to a replica set can be made using the
-the ``~pymongo.mongo_client.MongoClient`` method constructor, specifying
-one or more members of the set and optionally the replica set name.
-Any of the following connects to the replica set we just created:
+You can connect to an existing replica set by using the
+the ``~pymongo.mongo_client.MongoClient`` method constructor. Specify
```

Comment:
I confirmed that "method constructor" is an artifact of a search+replace I did. Should just be "constructor"

```suggestion
the ``~pymongo.mongo_client.MongoClient`` constructor. Specify
```